tests/transformations/state_fission_test.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+# The scope of the test is to verify state fission.
+# We use vector addition
+
+import dace
+import numpy as np
+import argparse
+import subprocess
+
+from dace.memlet import Memlet
+from dace.transformation import helpers
+
+
```

Comment:
Please use the `pytest` style for tests. i.e., `def test_blabla():` for each test and then in the `__main__` part call them in sequence.